### PR TITLE
Add a feature page for Editing Floating Disks

### PR DESCRIPTION
**Pull request by sleviim**
Changes proposed in this pull request:

- Add a feature page for Editing Floating Disks

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sleviim 
This pull request needs review by: @rollandf @danielerez

### CODE REVIEW

File: source/develop/release-management/features/storage/editing-floating-disks.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Editing Floating Disks
+category: feature
+authors: sleviim
+feature_name: Editing Floating Disks
+feature_modules: storage,engine,model
+feature_status: WIP 4.4
```
Comment (rollandf) [inline source/develop/release-management/features/storage/editing-floating-disks.md]: Why WIP? It is done, no?

File: source/develop/release-management/features/storage/editing-floating-disks.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+It is now possible editing disks from the 'Disks' tab, and also from the REST API, without prior attaching those disks to VMs, so they can remain floating.
+
+### User Experience
+
+A new 'Edit' button was added to the 'Disks' tab:
+
+![](/images/wiki/edit-floating-disks-button.png)
+
+
+Once the popup shows up, the user can edit the disk's non-VM
```
Comment (rollandf) [inline source/develop/release-management/features/storage/editing-floating-disks.md]: What about disks that are attached to VMs? Can we edit them also?

File: source/develop/release-management/features/storage/editing-floating-disks.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+dependent parameters, (such as it's alias and description), and also to extend its size:
+
+![](/images/wiki/edit-floating-image-disk.png)
+
+### REST
+
+A disk’s update can be done using the REST API.
```
Comment (rollandf) [inline source/develop/release-management/features/storage/editing-floating-disks.md]: Maybe:
A disk can be update using the REST API.

File: source/develop/release-management/features/storage/editing-floating-disks.md (hunk 1 above)
Comment (sleviim) [reply source/develop/release-management/features/storage/editing-floating-disks.md]: I thought it's a standard.
done

File: source/develop/release-management/features/storage/editing-floating-disks.md (hunk 2 above)
Comment (sleviim) [reply source/develop/release-management/features/storage/editing-floating-disks.md]: Yes we can, but there're fewer characteristics that can be edited.

File: source/develop/release-management/features/storage/editing-floating-disks.md (hunk 3 above)
Comment (sleviim) [reply source/develop/release-management/features/storage/editing-floating-disks.md]: done

**Pull request merged**